### DISCUSSION TITLE
How would I go about adding the seek feature from pull/674?

### DISCUSSION THREAD

**Discussion by Walht02** [jagrosh/MusicBot#1183] [Questions]
I'm looking to add the seek feature to JMusicBot. I saw someone had added the feature in this pull request:
https://github.com/jagrosh/MusicBot/pull/674

Would it be possible for me to get the changes from this pull request into my bot? Hardly ever used git or java. I tried cloning the master JMusicBot thing with the pull request and compiling it into a .jar, but that didn't work. Would it be possible for me to do this?

**Comment by sorasin** [comment]
i too would love to know this. dying to be able to skip through songs/albums to pick up where it was left off or skip segments.

**Comment by YJSoft** [comment]
You can(source code license is Apache 2.0 License).
But no support will provided if you do that. See README [Editing section](https://github.com/jagrosh/MusicBot#editing) for details.
